grisera_api/experiment/experiment_service_ontology.py

```diff
@@ -107,4 +107,24 @@ def delete_experiment(self, experiment_id: str):
         response = self.ontology_api_service.delete_instance(model_id, "Experiment", instance_label)
         if response["errors"] is not None:
             return ExperimentOut(experiment_name=instance_label, errors=response["errors"])
-        return ExperimentOut(experiment_name=response["label"])       
+        return ExperimentOut(experiment_name=response["label"])
+
+    def get_experiments(self):
+        """
+        Send request to ontology api to get experiments
+
+        Returns:
+            Result of request as list of experiments objects
+        """
+        model_id = 1
```

Comment:
Why is `model_id` explicitly set to 1?

Reply:
@mrwrob There is no `model_id` parameter in the base method, so we were asked to hard-code it for the time being

Reply:
model_id is set to 1, because in grisera framework there is not possibility to choose this argument, which is very important in our ontology_api. We cannot add this functionality in easy way and we don't have autorization to do so